src/pcp/atop/atop-daily.sh

```
@@ -63,7 +63,7 @@ do
     esac
 done
 
-: ${LOGPATH:=/var/log/atop}
+: ${LOGPATH:=$PCP_IMPORTLOG_DIR/atop}
```

Shellcheck (0.11.0)

[info] This default assignment may cause DoS due to globbing. Quote it.

(SC2223)